src/client/mac/handler/exception_handler.h

```diff
@@ -80,13 +80,15 @@ class ExceptionHandler {
   // |minidump_id| is a unique id for the dump, so the minidump
   // file is <dump_dir>/<minidump_id>.dmp.
   // |context| is the value passed into the constructor.
+  // |user_context| is the context of exception's thread
   // |succeeded| indicates whether a minidump file was successfully written.
   // Return true if the exception was fully handled and breakpad should exit.
   // Return false to allow any other exception handlers to process the
   // exception.
   typedef bool (*MinidumpCallback)(const char* dump_dir,
                                    const char* minidump_id,
-                                   void* context, bool succeeded);
+                                   void* context, 
+                                   breakpad_ucontext_t *user_context, bool succeeded);
```

Comment:
This is quite a breaking change.
In theory, unlike our crashpad fork, we still support building/linking against an upstream breakpad.

Reply:
for some reason they didn't want to provide thread context (probably it was just simpler)